specifications/api/swagger.yaml

```diff
@@ -4660,6 +4766,33 @@ paths:
         - *parametersWalletId
       responses: *responsesListAssets
 
+    post:
+      operationId: mintBurnAssets
+      tags: ["Assets"]
+      summary: Mint/Burn
+      description: |
+        <p align="right">status: <strong>stable</strong></p>
+
+        Mint and burn assets from the wallet.
```

Comment:
It would be worth explaining a little bit about how the script is made -- i.e. how policy keys are chosen.

Reply:
Slightly improved this with b245fe4f9.